### PR TITLE
Navigation bar fixes

### PR DESCRIPTION
**Pull request by nirvn**
@signedav , yep, this is truly meant for you :wink: this PR fixes a series feature list navigation bar issues we discussed.

### CODE REVIEW

File: src/qml/FeatureListForm.qml [hunk 1]
```diff
@@ -572,16 +572,19 @@ Rectangle {
 
   function hide()
   {
-    props.isVisible = false
-    focus = false
-    fullScreenView = qfieldSettings.fullScreenIdentifyView
+    props.isVisible = false;
```
Comment (suricactus) [inline src/qml/FeatureListForm.qml:575]: Do we move to semicolon syntax? I am pretty sure most of the codebase is without?

Comment (nirvn) [reply src/qml/FeatureListForm.qml:575]: I don't know if we have an agreement on this :) on my end, I try to add whenever I stumble on semicolon-less { functions } (but not for one-liners).

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 830233f886779160316afcbf0bae489010eb0c79:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1794-43db6ec526d89792bb75d60739370522f37826b8-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1794-43db6ec526d89792bb75d60739370522f37826b8-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1794-43db6ec526d89792bb75d60739370522f37826b8-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1794-43db6ec526d89792bb75d60739370522f37826b8-x86.apk)

**Review by signedav — APPROVED** [review]
Thanks.

**Pull request merged**